symbols/arm9/itcm.yml

```diff
@@ -0,0 +1,70 @@
+itcm:
+  versions:
+    - EU
+    - NA
+    - JP
+    - EU-ITCM
+    - NA-ITCM
+  address:
+    EU: 0x20B3CC0
+    NA: 0x20B3380
+    EU-ITCM: 0x1FF8000
+    NA-ITCM: 0x1FF8000
+  length:
+    EU: 0x4084
```

Comment:
> I don't know where that 0x4000 came from, since this special area in arm9.bin is 0x4080 bytes long.

Are you sure about this? `arm9.bin` does go a bit past 0x4000, but if you actually look in RAM while the game is running, the stuff starting at an offset of 0x4000 is all zeroes, whereas `arm9.bin` itself has some addresses there. RAM and `arm9.bin` only match up to a length of 0x4000, which is why I think we should mark the length as 0x4000.

(Also, even if we are going until the end of `arm9.bin`, the length should be 0x4078, not 0x4084)

Reply:
Okay, you're right, not everything past 0x20B3CC0 gets copied to the ITCM, only the first 0x4000 bytes. I'll change this.